README.md

```diff
@@ -204,7 +217,39 @@ Status | Error codes
     }
 }
 ```
-
+**Field descriptions**:
+* **payload** - base64 encoded string of the data items sent across
+* metadata.**businessId** - identifier for the name of the tax regime
+* metadata.**notableEvent** - a human-readable name for the type of submission
+* metadata.**payloadContentType** - the content type of the unencoded payload
+* metadata.**payloadSha256Checksum** - the SHA-256 checksum of the original payload prior to encoding
+* metadata.**nrSubmissionId** - globally unique identifier for this NRS submission
+* metadata.**userSubmissionTimestamp** - the date and time the user submitted the information
+* metadata.**identityData** - the identity of the individual making the submission
```

Comment:
Can you add a bit more detail on this item please. Some of the items within this data object will have to be optional as a business for example cannot have a DOB.

Reply:
For example if look at the vat-api here you can see which ones are: ter/app/uk/gov/hmrc/vatapi/models/NRSSubmission.scala#L52